index.css

```diff
@@ -0,0 +1,288 @@
+body {
+  background: darkblue;
+  height: 600vh;
+}
+
+.header-title {
```

Comment:
> Please change your header class header-title to header-section to make it more descriptions.

**Optional**
You can also write it in a different way to make it more understandable.
Ex: I think it would be better if you change your `header-title` class name to `header-section` to make it more descriptives.

Reply:
- [ ] You can also add emoji's to your comments they make it more human and friendly 👍🏿